### PR TITLE
bug1 + mising sum issue

### PR DESCRIPTION
**Pull request by Bukter**
+ symbol was missing
can you pls review

### CODE REVIEW

File: calci1.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ echo "Enter your choice"
 read ch
 
 case $ch in
-1)sum=`expr $n1 - $n2`
+1)sum=`expr $n1 + $n2`
```
Comment (Lakshmikanthwodeyar) [inline calci1.sh:16]: can u plz add a comment.

**Review by Lakshmikanthwodeyar — CHANGES_REQUESTED** [review]
plz look into my comments.

File: calci1.sh (hunk 1 above)
Comment (Bukter) [reply calci1.sh:16]: I hav implemented review comments,can you pls approve

**Review by Lakshmikanthwodeyar — APPROVED** [review]
Looks good to me. LGTM

**Pull request merged**